cross-compile.sh

```diff
@@ -19,7 +19,7 @@ function buildem() {
   done
 }
 
-GOARCHES="386 amd64 arm arm64 ppc64 ppc64le s390x mips mipsle mips64 mips64le"
+GOARCHES="386 amd64 arm arm64 ppc64 ppc64le s390x loong64 mips mipsle mips64 mips64le"
```

Comment:
I would revert this part. This'll require they use Go 1.19 (not sure if they do). But the other part of the change is safe.

Reply:
@yetist PTAL

Reply:
I still see it. Did you forget to push?

Reply:
Done.

Reply:
consider resolving this if addressed